query/query_test.go

```diff
@@ -38,6 +38,7 @@ func (s *TestSuite) SetUpSuite(t *C) {
 
 func (s *TestSuite) TestFingerprintBasic(t *C) {
 	var q string
+	query.ReplaceNumbersInWords = false
```

Comment:
Since this var is false by default, these lines shouldn't be needed. Just need `query.ReplaceNumbersInWords = true` in the test case for this hack (and probably `defer func() { query.ReplaceNumbersInWords = false }()` too).